### DISCUSSION TITLE
Automate hide show based on status of device.

### DISCUSSION THREAD

**Discussion by askthomas** [PiotrMachowski/Home-Assistant-Lovelace-Local-Conditional-card#19] [Ideas]
If would be nice if can hida show card based on devce. Ex, power to 3D printer = off hide card for printer, when turn on, show all extra in 3D printer has when powerd on.

Also TV nad power save, I today power of whole TV/Reciver/Appel TV/DVD/PS 5 etc, when not using, then the will be unreaceble in HAS, would be relay cool to hode them with this card when powerd off.

/T

**Comment by PiotrMachowski** [comment]
I think you might want to use [built-in conditional card](https://www.home-assistant.io/dashboards/conditional/) instead

**Comment by askthomas** [comment]
Aha, thanks. I missed that one.

/Thomas
